### DISCUSSION TITLE
Help needed with Nordvpn

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#1958] [Q&A] [answered]
Hi, 

To launch the container I do:
```
sudo docker run --cap-add=NET_ADMIN -d \
              -v ~/data_transmission:/data \
              -e OPENVPN_PROVIDER=NORDVPN \
              -e OPENVPN_CONFIG=FR \
              -e OPENVPN_USERNAME=myusername\
              -e OPENVPN_PASSWORD=mypassword  \
              --log-driver json-file \
              --log-opt max-size=10m \
              -e LOCAL_NETWORK=****** \
              -p 9091:9091 \
              haugene/transmission-openvpn
```

Then I have this in Nginx to access to web GUI:
```

server {

    server_name *****;

    location / {
proxy_pass http://localhost:9091/;
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Host $host;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header Connection upgrade;
proxy_set_header Accept-Encoding gzip;

    }
```

When I run the first command and then "docker ps" there is nothing. Sometimes there is something but then it disappears. 
When I run "ip addr" I get three entries: lo, venet0 and docker0 which one should be set to the LOCAL_NETWORK? I don’t really understand how to get this working.

I'm kind of new to these concepts and would really appreciate it if someone could explain me what I'm doing wrong. I have read the documentation but don't really know what to do with it. Thank you.

**Comment by edgd1er** [comment, marked as answer]
Hi @spouradrix , 
you may find some information here: https://haugene.github.io/docker-transmission-openvpn/provider-specific/#nordvpn
never used openvpn_config in conjonction with nordvpn_XXXX attributes, so I'm not sure what is wrong.
I use only NORDVPN_XXX + OPENVPN_OPTIONS env.

docker logs should show some information about what is going on.

**Reply by pkishino** [reply]
NORDVPN uses specific provider options that have to be used

**Reply by ghost** [reply]
I solved it. OPENVPN_CONFIG=FR caused the software to look for france.ovpn instead of looking through NORDVPN's servers.. NORDVPN_COUNTRY=XX made it work.
